docs/book/src/developers/releasing.md

```diff
@@ -106,6 +112,10 @@ Using [the above example PR](https://github.com/kubernetes/k8s.io/pull/4284), to
 ### Release in GitHub
 
 - Manually format and categorize the release notes
+- Ensure that the promoted release image is live. For example:
+```sh
+$ docker pull registry.k8s.io/cluster-api-azure/cluster-api-azure-controller:v1.2.3
```

Comment:
```suggestion
$ docker pull registry.k8s.io/cluster-api-azure/cluster-api-azure-controller:${RELEASE_TAG}
```

if you run this from the same shell as earlier you can copy paste the command without needing to change it